src/main/java/eu/execom/pomodoroTeam/controllers/TeamController.java

```diff
@@ -76,13 +82,15 @@ public TeamController(TeamRepository teamRepository, UserRepository userReposito
      * @return
      */
     @RequestMapping(method = RequestMethod.PUT, value = "/{id}")
-    public ResponseEntity<?> updateTeam(@PathVariable Long id, @RequestBody TeamDto teamDto) {
+    public ResponseEntity<TeamDto> updateTeam(@PathVariable Long id, @RequestBody TeamDto teamDto) {
         TeamEntity team = teamRepository.getOne(id);
         if (team.getName() != null) {
             team.setName(teamDto.getName());
         }
         teamRepository.save(team);
-        return new ResponseEntity<>(team, HttpStatus.OK);
+        TeamDto teamDto2 = new TeamDto();
+        teamDto2.setName(team.getName());
```

Comment:
It is better if you name variables semantically, e.g., `updatedTeamDto`. Also, it would be more correct if you map the updated TeamEntity to the resulting TeamDto, instead of creating a new instance and just setting the name. e.g.,
```
TeamDto updatedTeamDto = mapper.map(team);
return new Responseentity(updatedTeamDto, HttpStatus.OK);
```